README.md

```diff
@@ -15,7 +15,30 @@ $ npm i react-with-hooks --save
 
 ## Example
 
+You can use `react-with-hooks` as a polyfill; in this case, when you later transition to native React Hooks you will only need to replace `'react-with-hooks/polyfill'` with `'react'` in your import statments:
+
+```javascript
+import React, { useState, useEffect } from 'react-with-hooks/polyfill';
```

Comment:
What I understand a polyfill is that we monkey patch the React, so that people can use `react` directly after import `react-with-hooks/polyfill`:

```javascript
import 'react-with-hooks/polyfill'; // monkey patch React
import React from 'react'; // Use the native react
```

Reply:
Yes, that’s more correct, but I couldn’t find a way to do that. A similar solution would be that users could alias `react` to `react-with-hooks/polyfill` in their Webpack/Rollup/etc config. This would be best because no changes at all (except removing the alias) would be needed when moving to React 16.8, but again, I couldn’t find a way to do that while ensuring that `react-with-hooks` can still import the “real” `'react'`.

I’m happy if you’d prefer to use a different name instead of “polyfill” :)